runtime/amplitude/src/lib.rs

```diff
@@ -1117,11 +1120,7 @@ impl oracle::Config for Runtime {
 	type WeightInfo = oracle::SubstrateWeight<Runtime>;
 	type DataProvider = DataProviderImpl;
 	#[cfg(feature = "runtime-benchmarks")]
-	type DataFeedProvider = DataFeederBenchmark<
-		oracle::OracleKey,
-		oracle::TimestampedValue<UnsignedFixedPoint, Moment>,
-		Self::AccountId,
-	>;
+	type DataFeeder = MockDataFeeder<Self::AccountId,Moment>;
```

Comment:
Could we use the `MockDataFeeder` that is used in Spacewalk's oracle pallet altogether or is there an issue with re-using that?

Reply:
The problem is that it makes uses of some globally defines variables [COINS and LOCK](https://github.com/pendulum-chain/spacewalk/blob/13d25ae642cf43a65ebcb6028f5f0bc5b4e92160/pallets/oracle/src/testing_utils/mock_collector.rs#L105-L127)

Reply:
But does it matter if we just import it from the crate itself, like [here](https://github.com/pendulum-chain/spacewalk/blob/13d25ae642cf43a65ebcb6028f5f0bc5b4e92160/testchain/runtime/src/lib.rs#L514)?

Reply:
Actually you are right! I'll remove the mock here.